frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -332,6 +333,13 @@ export const funnelLogic = kea<funnelLogicType<TimeStepOption>>({
                 return stepsDropdown
             },
         ],
+        totalConversionRate: [
+            () => [selectors.stepsWithCount],
+            (stepsWithCount) =>
+                stepsWithCount.length > 1
```

Comment:
Not sure if this is the right way to compute the total percentage or there is a less hacky way of doing this?